apps/client/src/app/pages/accounts/create-or-update-account-dialog/create-or-update-account-dialog.component.ts

```diff
@@ -4,14 +4,24 @@ import {
   Inject,
   OnDestroy
 } from '@angular/core';
-import { FormBuilder, FormGroup, Validators } from '@angular/forms';
+import {
+  AbstractControl,
+  FormBuilder,
+  FormGroup,
+  ValidatorFn,
+  Validators
+} from '@angular/forms';
 import { MAT_DIALOG_DATA, MatDialogRef } from '@angular/material/dialog';
 import { CreateAccountDto } from '@ghostfolio/api/app/account/create-account.dto';
 import { UpdateAccountDto } from '@ghostfolio/api/app/account/update-account.dto';
 import { DataService } from '@ghostfolio/client/services/data.service';
-import { Subject } from 'rxjs';
+import { Observable, Subject } from 'rxjs';
+import { map, startWith } from 'rxjs/operators';
 
-import { CreateOrUpdateAccountDialogParams } from './interfaces/interfaces';
+import {
+  CreateOrUpdateAccountDialogParams,
+  Platform
```

Comment:
Can you reuse `import { Platform } from '@prisma/client';` instead of your own interface?

Reply:
yes sure, bd2b36622d4ccea0b9086ba6c7ba4f66be2010b3

because of the property "url" i had to map the infos from
`this.dataService.fetchInfo();`

with the following adjustment

```
 this.platforms = platforms.map((platform) => {
      return { ...platform, url: '' };
    });
```